loaders/gtdb/gtdb_config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "LOAD_VERSION": 207,
+  "KBASE_COLLECTION": "gtdb",
+  "LOAD_FILES": [
+    "~/SCIENCE/taxonomy/gtdb/bac120_taxonomy_r207.tsv",
+    "~/SCIENCE/taxonomy/gtdb/ar53_taxonomy_r207.tsv"
+  ]
+}
```

Comment:
I would strongly recommend something other than JSON for the config format - it sucks to edit for a human and doesn't have comments. YAML or TOML are much better

Reply:
👍